mmeds/tests/unit/test.py

```diff
@@ -159,5 +138,33 @@ def remove_users(users_added):
             remove_user(fig.TEST_USER_0, testing=testing)
 
 
+def main():
+    tests = sys.argv[1:]
+    log = 'log' in tests
+    if log:
+        tests.remove('log')
+
+    setup = 'setup' in tests
+    if setup:
+        tests.remove('setup')
+
+    cleanup = 'cleanup' in tests
+    if cleanup:
+        tests.remove('cleanup')
+
+    if len(tests) == 0:
```

Comment:
why not ```if tests:```

Reply:
Uh I did not add that line. I'll change it though. If I'm gonna be modifying that file I might as well fix it all